RadarSDK/remodel/RadarBeacon.m

```diff
@@ -0,0 +1,132 @@
+/**
+ * This file is generated using the remodel generation script.
+ * The name of the input file is RadarBeacon.value
+ */
+
+#if !__has_feature(objc_arc)
+#error This file must be compiled with ARC. Use -fobjc-arc flag (or convert project to ARC).
+#endif
+
+#import "RadarBeacon.h"
+#import "RadarCollectionAdditions.h"
+
+#pragma clang diagnostic push
+#pragma GCC diagnostic ignored "-Wobjc-designated-initializers"
+
+#define RMParameterAssert(condition) NSCParameterAssert((condition))
+
+NS_ASSUME_NONNULL_BEGIN
+
+@implementation RadarBeacon
+
+- (nullable instancetype)initWithRadarJSONObject:(nullable id)object {
+    if (!object || ![object isKindOfClass:[NSDictionary class]]) {
+        return nil;
+    }
+    NSDictionary *dictionary = (NSDictionary *)object;
+    if ((self = [super init])) {
+        if (!dictionary[@"_id"]) {
+            return nil;
```

Comment:
if the value in the dictionary is null while the property is nonnull, we just return nil from this init method